src/Input/Input.js

```diff
@@ -89,6 +129,22 @@ module.exports = kind(
 	},
 
 	/**
+	* @private
+	*/
+	components: [
+		{name: 'TooltipDecorator', kind: TooltipDecorator, autoShow: false, components: [
```

Comment:
I don't know if we necessarily need to specify an explicit name here. If so, we should be consistent with our naming convention and make it lower camel case.